tests/venv/apt-base/apt.yaml

```diff
@@ -0,0 +1 @@
+base: artful
```

Comment:
Does apt.yaml replace apt.txt? Or is it an additional file with only one supported key? What's the plan for transition here?

Reply:
Yeah, I think it should replace apt.txt. I was trying to get it to work with just this one key now.